src/views/Nft/market/Collection/Items/CollectionNfts.tsx

```diff
@@ -17,33 +22,65 @@ interface CollectionNftsProps {
 
 const CollectionNfts: React.FC<CollectionNftsProps> = ({ collection, sortBy = 'updatedAt' }) => {
   const { address } = collection
+  const [allPancakeBunnyNfts, setAllPancakeBunnyNfts] = useState<NftToken[]>(null)
   const checksummedAddress = getAddress(address)
   const nfts = useNftsFromCollection(checksummedAddress)
   const dispatch = useAppDispatch()
 
+  const isPBCollection = address === pancakeBunniesAddress
+
   useEffect(() => {
     dispatch(fetchNftsFromCollections(checksummedAddress))
   }, [checksummedAddress, dispatch])
 
-  if (!nfts) {
+  useEffect(() => {
+    const fetchPancakeBunnies = async () => {
```

Comment:
Given it's so specific to one collection - might be worth putting this effect in its own hook, returning `allPancakeBunnyNfts`?